### ISSUE TITLE
Trigger character or manual trigger?

### ISSUE THREAD

**Issue by chuan2984** [SergioRibera/cmp-dotenv#3]
Hey there, i have been searching for a completion source like this.
I work in a codebase that contains 100+ envs, and almost any letter would be a hit. Is there a way to set this so that it only triggers on all snake case captialized chracters? Or is there a way to set a manual trigger? 
Thanks!

**Comment by SergioRibera** [comment]
You're right, I did it this way so I could get the completion every time, I think customizing it might be a good idea

**Comment by chuan2984** [comment]
Any pointer in customizing it, pretty please? im relatively new. Google around for a bit couldnt find much

**Comment by ph1losof** [comment]
Hi @chuan2984 if you are still interested, I have made a plugin which provides all-in-one toolkit to work with `.env` files and environment variables. You can check it out here: https://github.com/philosofonusus/ecolog.nvim

It allows you to add custom trigger patterns for different file-types to avoid such issue. It has both `nvim-cmp` and `blink-cmp` support, and it implements this specific feature here: 
https://github.com/philosofonusus/ecolog.nvim?tab=readme-ov-file#-custom-providers

**Comment by chuan2984** [comment]
@philosofonusus a great plugin, just downloaded it, its working flawlessly, exactly what I lacked. Thanks!
